### DISCUSSION TITLE
Tag-Based File Manager

### DISCUSSION THREAD

**Discussion by ItzPvPNinja** [linuxmint/discussions#477] [Ideas]
Lately I found a very nice project about a file manager that does not use simple folder structures but a rather smart tag-organisation. Because I'm not too good at explaining it, please look at the repo ( https://github.com/TagStudioDev/TagStudio ) or the YouTube video presenting the concept ( https://www.youtube.com/watch?v=wTQeMkYRMcw ). 
While this project is by far not finished yet, i think the concept is definitely worth noticing and maybe you could implement something in nemo that works similar. 
Please keep in mind that this is not about "simple tags" as in Apples "Finder" but about a structure concept which is able to organize files in a more structured way, folders could ever do. 
Additionally I like the concept, that this Tag-Organization can still be used like the traditional Folder-Structure but adds the possibility to do more than just that.

**Comment by Jopp-gh** [comment]
Subscribed.
Here you can read about my proposal
https://github.com/orgs/linuxmint/discussions/430
Here a quick mockup:
![333147384-bfceeb4e-6ade-4aeb-a613-1280262d69f8](https://github.com/linuxmint/discussions/assets/48620536/cb32a71d-a87d-4463-bed6-2b7589150d47)

**Reply by ItzPvPNinja** [reply]
that looks very promising. Especially the smarter often-visited thing might be powerful. 
Nevertheless the extended tag functionality I described above (referring to the YT video) seems like a good addition to that.

**Comment by cplittlfield** [comment]
It's a cool concept, tags.  If implemented, I can see that this feature would be useful for organizing text files, such as research studies that involve a number of different topics, and not just images.

After Pix stopped being useable, I started storing images in multiple categories by creating hard links to the same file in different directories (a feature of ext4 that I _really_ like).  The ability to add searchable tags would also be useful, since many people are afraid of the command line and don't want to bother with creating hard links.  As far as images are concerned, the ability to edit and search exif data might be enough, but this is a feature that needs to be generalized to files beyond simply images.

**Comment by Jopp-gh** [comment]
I think tags are overlooked: after more than a decade and more, interfaces integrate tags more often, but it seems people are still not ready to organize data with tags, but prefer instead outdated and very manual clicking through a ton of folders. 

_Please add tags as navigation method to Nemo!_

I want to suggest the devs of Mint to work on a prototype so you would see by yourself how powerful tags could be. Please see the links of the OP and my proposal too. 

All the recent graphical redesign of Mint is nice, but only a secondary thing, bc themes are like clothes/"skins" only, nothing essential but very "variable" and should be more a personal thing than something the devs have to worry.
